components/brave_wallet_ui/stories/locale.ts

```diff
@@ -882,6 +882,22 @@ provideStrings({
   braveWalletTransactionApproveUnlimited: 'Unlimited',
   braveWalletApprovalTransactionIntent: 'approve',
 
+  // Transaction Type Names
+  braveWalletTransactionTypeNameErc20Approve: 'Approve',
+  braveWalletTransactionTypeNameSafeTransferFrom: 'Safe Transfer',
+  braveWalletTransactionTypeNameTokenTransfer: 'Token transfer',
+  braveWalletTransactionTypeNameNftTransfer: 'Transfer NFT',
+  braveWalletTransactionTypeNameForwardFil: 'Forward FIL',
+  braveWalletTransactionTypeNameOther: 'Other',
+  braveWalletTransactionTypeNameCompressedNftTransfer:
+    'Compressed NFT Transfer',
+  braveWalletTransactionTypeNameSignAndSendDappTransaction:
+    'Sign and Send DApp transaction',
+  braveWalletTransactionTypeNameSignDappTransaction: 'Sign DApp Transaction',
+  ['braveWalletTransactionTypeNameSplTokenTransfer' +
+  'WithAssociatedTokenAccountCreation']:
+    'SPL Token Transfer With Associated Token Account Creation',
```

Comment:
Should that be just `braveWalletTransactionTypeNameSplTokenTransferWithAssociatedTokenAccountCreation`?
And/or maybe some shorter name?

Reply:
Yeah it will equal `braveWalletTransactionTypeNameSplTokenTransferWithAssociatedTokenAccountCreation`. I used concatenation to break-up the long name into multiple lines to avoid presubmit errors. Do you know if there is a way to suppress pre-submit errors for a single line?